utilities/build.sh

```diff
@@ -114,4 +124,7 @@ fi
 
 # exit if configuration failed with errors
 if [ ! $? -eq 0 ]; then exit 1; fi
+if [[ ${clean_flag} = "true" ]]; then
+  "${BUILD_TOOL}" clean;
```

Comment:
why do we need `;` at the end of line?
it is not needed when invoking for compilation (`${BUILD_TOOL}" -j8 all`)

Reply:
fixed